### PR TITLE
Remove amd64 assumption in release.sh, cross and tgz scripts

### PR DESCRIPTION
**Pull request by mlaventure**
Signed-off-by: Kenfe-Mickael Laventure mickael.laventure@gmail.com
## 

This was breaking generation of the `tgz` archive on arm

ping @andrewhsu @tianon @tiborvass @vieux

### CODE REVIEW

**Comment by tianon** [comment]
Fair enough; LGTM :+1:

**Comment by vieux** [comment]
LGTM

File: hack/release.sh [hunk 1]
```diff
@@ -258,7 +258,7 @@ release_build() {
 
 # Upload binaries and tgz files to S3
 release_binaries() {
-	[ -e "bundles/$VERSION/cross/linux/amd64/docker-$VERSION" ] || {
+	[ "$(find bundles/$VERSION -path 'bundles/$VERSION/cross/*/*/docker-$VERSION)" = "" ] || {
```
Comment (andrewhsu) [inline hack/release.sh]: does this single quote need a corresponding closing one?

**Review by andrewhsu — COMMENTED** [review]
check single quotes

**Comment by tianon** [comment]
heh, good catch! :+1:

File: hack/release.sh (hunk 1 above)
Comment (mlaventure) [inline hack/release.sh]: yes, just realized, this is the only part of the script I couldn't test :angel:

**Comment by mlaventure** [comment]
Fixed

**Comment by andrewhsu** [comment]
LGTM

**Review by vieux — APPROVED** [review]
LGTM

**Comment by thaJeztah** [comment]
@mlaventure @tiborvass does this mean we can re-enable debian for arm again?

**Comment by mlaventure** [comment]
@thaJeztah this only fixed the static binaries issue, nothing related with the deb packaging.

We would need to test, but since nothing has change, I'd assume they would still overwrite each other.

**Pull request merged**